facebook/__init__.py

```diff
@@ -96,6 +143,8 @@ def __init__(
         proxies=None,
         session=None,
         app_secret=None,
+        logger=None,  # expected to contain logging level's debug() and error()
```

Comment:
This comment doesn't make it clear what this parameter is supposed to contain. The "API Reference" in the documentation should also be updated when new parameters are added.